Library/Formula/libfreenect2.rb

```diff
@@ -0,0 +1,128 @@
+class Libfreenect2 < Formula
+  desc "Drivers/Example for Kinect V2"
+  homepage "https://github.com/OpenKinect/libfreenect2"
+  url "https://github.com/ybott/libfreenect2/archive/v0.0.1-alpha.tar.gz"
```

Comment:
This release *needs* to come from the upstream repository, not your own fork. Only an official upstream release is acceptable I'm afraid.